pages/index.tsx

```diff
@@ -1,118 +1,572 @@
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <h2 className={`mb-3 text-2xl font-semibold`}>
-            Templates{" "}
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Discover and deploy boilerplate example Next.js&nbsp;projects.
-          </p>
-        </a>
-
-        <a
-          href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template-tw&utm_campaign=create-next-app"
-          className="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <h2 className={`mb-3 text-2xl font-semibold`}>
-            Deploy{" "}
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50 text-balance`}>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
+      <section className="bg-center flex">
+        <div className="container mx-auto justify-between items-center flex">
+        <div className="grid grid-cols-2 lg:grid-cols-6 gap-6 xl:gap-8  py-8 xl:py-14 px-8 xl:px-14 bg-white shadow-xl rounded-md -mt-0 sm:-mt-36 mx-0 xl:mx-10">
+          <div>
+              <img className="h-16 max-w-full rounded-lg" src="./assets/Medmonk-Network.png" alt="" />
+              <hr className=" h-0.5 w-24 bg-brownCust mb-2 mt-4"/>
+              <h3 className="font-semibold text-hBlack mb-2 text-lg">Medmonk Network:</h3>
+              <p>Connected and contracted with 98% of specialty pharmacies across the U.S.</p>
+          </div>
+          <div>
+              <img className="h-16 max-w-full rounded-lg" src="./assets/Support-Provided.png" alt="" />
+              <hr className=" h-0.5 w-24 bg-brownCust mb-2 mt-4"/>
+              <h3 className="font-semibold text-hBlack mb-2 text-1xl">Support Provided:</h3>
+              <p>2.3 million patients supported with $2 billion in claims</p>
+          </div>
+          <div>
+              <img className="h-16 max-w-full rounded-lg" src="./assets/Brand-Loyalty.png" alt="" />
+              <hr className=" h-0.5 w-24 bg-brownCust mb-2 mt-4"/>
+              <h3 className="font-semibold text-hBlack mb-2 text-lg">Brand Loyalty:</h3>
+              <p>Brands stay with us for an average of 7 years</p>
+          </div>
+          <div>
```

Comment:
@vishalsolanki40 => Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element

Reply:
Removed